data-access-layer/eventmesh/LockManager.js

```diff
@@ -94,14 +94,14 @@ class LockManager {
    * @param {string} [lockDetails.lockedResourceDetails.operation] - Operation type who is acquiring the lock. ex: backup
```

Comment:
Can you please add `plan` also in jsdoc as optional param

Reply:
done